src/signals/incident-management/index.js

```diff
@@ -128,6 +128,10 @@ const IncidentManagement = () => {
             <Route path={routes.standardTexts}>
               <Route index element={<StandardTextsAdmin />} />
               <Route path={`:id`} element={<StandardTextsDetail />} />
+              <Route
```

Comment:
this should be part of the detail page...it currently has its own nested routes for reusability purposes cause new :id/edit and :id/subcategories are all within the detail space